app/views/dashboards/_modal.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="modal fade new-call" tabindex="-1" role="dialog">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-body">
+        <div class="calls-layout">
+          <button type="button" class="close close-terms" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+          <h4 class="modal-title calls-request">
```

Comment:
Since everyone (patient submit/cancel pledge) is trying to populate the modal, I am going to have it clear at the ```<div class="modal-content">``` for consistency. In the future, every new modal popup needs to follow the same rules as calls#new